backend/substrapp/migrations/0004_key_checksum.py

```diff
@@ -0,0 +1,23 @@
+from django.apps import apps
+from django.db import migrations, models
+from django.db.models import F
```

Comment:
looks like is not used

Reply:
Fixed in https://github.com/SubstraFoundation/substra-backend/pull/342/commits/9186a1de0490d28af75f6dd9c5228e4a2c217ff5